armi/settings/fwSettings/globalSettings.py

```diff
@@ -129,6 +130,18 @@ def defineSettings() -> List[setting.Setting]:
             description="Number of CPUs to request on the cluster",
             schema=vol.All(vol.Coerce(int), vol.Range(min=1)),
         ),
+        setting.Setting(
+            CONF_INITIALIZE_BURN_CHAIN,
+            default=True,
+            label="Initialize Burn Chain",
+            description=(
+                f"This setting is paired with the `{CONF_BURN_CHAIN_FILE_NAME}` setting. "
+                f"When enabled, this will initialize the burn-chain on initializing the case and "
+                f"is required for running depletion calculations where the transmutations and decays "
+                f"are controlled by the framework. If an external software, such as ORIGEN, contains "
+                f"data for the burn-chain already embedded then this may be disabled."
+            ),
+        ),
```

Comment:
Not necessary, but maybe you want to add a schema to only allow booleans?

Reply:
Implemented.